### ISSUE TITLE
MicaForEveryone not opening at all

### ISSUE THREAD

**Issue by Cubifyy** [MicaForEveryone/MicaForEveryone#326]
[no] This is not about other apps being displayed weird or etc.
[no] This is not about rendering issues that caused by a bug in the Windows.
[yes] I've read [FAQ](https://github.com/MicaForEveryone/MicaForEveryone/wiki/FAQ).
[yes] I've checked other issues and this is not a duplicate.

**Describe the bug**

The app will not open, I checked Event Viewer and it's showing errors pertaining to MicaForEveryone

**To Reproduce**
1. Opening the app

**Expected behavior**
The app to open and function properly

**Screenshots**
Event Viewer log saying "Application Error"
![image](https://github.com/MicaForEveryone/MicaForEveryone/assets/122413885/9b675b10-2323-477f-b6d1-2e736544f741)

Event Viewer log saying ".NET Runtime"
![image](https://github.com/MicaForEveryone/MicaForEveryone/assets/122413885/fe57673b-4d02-4c27-b310-94555cbb2b8c)

**Windows Version**
Windows 11 Built 22621.1992

**Additional context**

**Comment by dongle-the-gadget** [comment]
My guess is due to some WinRT registration-free manifest error. Which version did you use (ZIP, EXE installer, MSIX)?

**Comment by Cubifyy** [comment]
I used the .exe file

**Comment by dongle-the-gadget** [comment]
Right. I've seen some other person in another issue about the Inno Setup installer not working.

**Comment by Cubifyy** [comment]
Could I fix this by using another version e.g. ZIP, MSIX?

**Comment by dongle-the-gadget** [comment]
Some people say it could. IMO MSIX is the most reliable option.

**Comment by Cubifyy** [comment]
Alright I will try it

**Comment by xmine64** [comment]
Have you installed Visual C++ Runtime v141?

**Comment by zethriller** [comment]
I have the same issue and it started happening a few days ago.
Current status: KERNELBASE.dll related error in the application event log (same as op)
Exe (1.3.1.2): no change.
Zip (1.3.1.2) to a different location: no change.
Reinstalled VC++ Runtime: no change
Uninstalled last windows update kb since dates more or less match (2023-07 Cumulative update preview for windows 11 version 22h2 for x64-based systems (kb5028254)): no change
Tried installing MSIX, well... i'm certainly doing something wrong.
```
PS > Add-AppxPackage .\MicaForEveryone-x64-Release.msix
Add-AppxPackage : Deployment failed with HRESULT: 0x800B0109, A certificate chain processed, but terminated in a root certificate which is not trusted by the trust provider.
error 0x800B0109: The root certificate of the signature in the app package or bundle must be trusted.
NOTE: For additional information, look for [ActivityId] bd86f62d-c226-000b-b9e0-89bd26c2d901 in the Event Log or use the command line
Get-AppPackageLog -ActivityID bd86f62d-c226-000b-b9e0-89bd26c2d901
At line:1 char:1
+ Add-AppxPackage .\MicaForEveryone-x64-Release.msix
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (...\Mi...64-Release.msix:String) [Add-AppxPackage], Exception
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand
```

I'm stuck here as much as you are...

**Comment by dongle-the-gadget** [comment]
@zethriller as for MSIX, you need to follow this guide: https://github.com/MicaForEveryone/MicaForEveryone/wiki/Installation#msix-file

**Comment by zethriller** [comment]
Thanks, i just did that but i have the same error with the msix.
I don't see anything named Minusium in the certificate chain, just MicaForEveryone.

![image](https://github.com/MicaForEveryone/MicaForEveryone/assets/35899920/1f4cf9c0-7104-45d0-ae1b-dce4dfa82df9)

**Comment by dongle-the-gadget** [comment]
Oh. Install the MicaForEveryone certificate then.

**Comment by zethriller** [comment]
I did but still refuses to open.
The only way to install it is to use `Add-AppxPackage -AllowUnsigned` command in powershell, then it runs fine.
For anyone reading this, this is for testing purpose **only**, not something you should do.

**Comment by xmine64** [comment]
Installing custom certificate is not something you should do either, but well there's nothing else we can do.
